### ISSUE TITLE
A11y_VisualStudioCodeClient_Debugging_Configure 'launch.json'_Screen Reader:Color and shape are the only way to denote Breakpoint in editor window

### ISSUE THREAD

**Issue by VijayaBharathyA** [microsoft/vscode#159547]
### GitHub Tags:
#A11yMAS; #A11yTCS; #A11ySev2; #DesktopApp; #Win32; #Visual Studio Code Client; #BM-VisualStudioCodeClient-Win32-Aug2022; #WCAG1.3.3; 

### Environment and OS details:
Visual Studio Code Version: 1.70.2 (user setup)
Windows 11 Pro version 21H2 (OS Build 22000.856)

### Repro Steps:

1.Open Visual studio code editor.
2.Using "Ctrl+Shift+D" key navigate to the debug side view bar option.
3.Create a breakpoint by using this https://code.visualstudio.com/docs/editor/debugging#_breakpoints
4.Verify that shape and color is only way to denote breakpoint or not

### Actual Result:
Color and shape are the only way to denote Breakpoint in editor window 

### Expected Result:
Alternate text to indicate breakpoints should be present in addition to color and shape.

### Attachment:
![Sensory](https://user-images.githubusercontent.com/100189407/187412045-f9057e8b-098e-4735-8b56-eab47b7015b3.jpg)

**Comment by isidorn** [comment]
Breakpoints are shown in the editor glyph margin on the left/
To learn more about this please read these docs https://code.visualstudio.com/docs/editor/debugging

There is text in the breakpoints view. So the user can F1 > focus breakopints view to get text about this. Also on hover in the glyph there is text.

Closing as invalid

**Comment by VijayaBharathyA** [comment]
![159547](https://user-images.githubusercontent.com/100189407/188546352-39e842c2-e6f4-457b-a11b-8b37636db2d1.gif)
